docs/sources/installation/windows.md

```diff
@@ -21,18 +21,10 @@ installation.
 
 **Important:** After you've downloaded the zip file and before extracting it, make sure to open properties for that file (right-click Properties) and check the `unblock` checkbox and `Ok`.
 
-The zip file contains a folder with the current Grafana version. Extract
-this folder to anywhere you want Grafana to run from.  Go into the
-`conf` directory and copy `sample.ini` to `custom.ini`. You should edit
-`custom.ini`, never `defaults.ini`.
-
-The default Grafana port is `3000`, this port requires extra permissions
-on Windows. Edit `custom.ini` and uncomment the `http_port`
-configuration option (`;` is the comment character in ini files) and change it to something like `8080` or similar.
-That port should not require extra Windows privileges.
-
-Default login and password `admin`/ `admin`
+The zip file contains a folder with the current Grafana version. Extract this folder to anywhere you want Grafana to run from.  Go into the
+`conf` directory and copy `sample.ini` to `custom.ini`. You should edit `custom.ini`, never `defaults.ini`.
 
+The default Grafana port is `3000`, this port requires extra permissions on Windows. Edit `custom.ini` and uncomment the `http_port` configuration option (`;` is the comment character in ini files) and change it to something like `8080` or similar. That port should not require extra Windows privileges.
```

Comment:
Is any of this actually necessary? I was able to install with the msi installer and then it popped open localhost:3000 with no problem.

Reply:
It was earlier at least. Maybe msi installer adds a firewall exception or fixes the extra permission? Maybe it's problem when using zip file even though I haven't seen that, but can try it out tomorrow again. Or maybe it's related to older versions of Windows?

Reply:
@marefr , would you test that, then?

Reply:
Updated.